jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/service/impl/JvmCrudServiceImpl.java

```diff
@@ -359,7 +360,14 @@ public int removeTemplate(final String jvmName, final String templateName) {
     public List<Jvm> getJvmsByGroupName(String groupName) {
         final Query q = entityManager.createNamedQuery(JpaJvm.QUERY_GET_JVMS_BY_GROUP_NAME);
         q.setParameter(JpaJvm.QUERY_PARAM_GROUP_NAME, groupName);
-        return buildJvms(q.getResultList());
+        List<Jvm> listOfJvms=buildJvms(q.getResultList());
```

Comment:
Can you just sort via the name query ? 

e.g. in JpaJvm

    @NamedQuery(name = JpaJvm.QUERY_GET_JVMS_BY_GROUP_NAME, query = "SELECT j FROM JpaJvm j WHERE j.groups.name = :groupName ORDER BY j.name"),